gp-includes/cli.php

```diff
@@ -41,7 +41,7 @@ function __construct() {
 		_deprecated_function( 'GP_CLI::__construct', '', 'WP_CLI_Command' );
 
 		global $argv;
-		if ( gp_array_get( $_SERVER, 'HTTP_HOST' ) ) {
+		if ( wp_unslash( gp_array_get( $_SERVER, 'HTTP_HOST' ) ) ) {
```

Comment:
Just added it to all the locations for completeness.  I'll remove it from the PR.